assets/airports/eddh.json

```diff
@@ -1,844 +1,1435 @@
             "icao": "RARUP",
             "name": "Rarup",
+            "suffix": {"05":"05", "15":"15", "23":"23", "33":"33"},
             "entryPoints": {
-                "M748": ["BKD", ["BUMIL", "A230"], ["ABMAL", "A150"]]
+                "RARUP": []
             },
-            "body": [["RARUP", "A100|S250"]],
+            "body": ["RARUP"],
             "rwy": {
-                "05": [["DH230", "S220"], "DH235", "DH255", "DH256", "DH257", "DH025", "ANEXI"],
-                "15": [["DH410", "S220"], "DH412", "DH416", "DH455", "DH456", "DH457", "DH458"],
-                "23": [["DH600", "S220"], "DH611", "DH615", "DH655", "DH656", "DH657", "DH658"],
-                "33": [["DH829", "S220"], "DH830", "DH835", "DH854", "DH855", "DH856", "DH857", "DH858"]
+                "05": [["DH230", "S220"], "DH231", "DH232", "DH233", "DH234", ["DH235", "S220"], "DH255", "DH256", "DH257", "DH025", ["ANEXI", "S170+"]],
+                "15": [["DH417", "S220"], ["DH412", "S220"], "DH413", "DH414", ["DH416", "S220"], "DH455", "DH456", "DH457", "DH458", ["DEVOD", "S170+"]],
+                "23": [["DH600", "S220"], ["DH611", "S220"], "DH612", "DH613", "DH614", ["DH615", "S220"], "DH655", "DH656", "DH657", "DH658", ["PISAS", "S170+"]],
+                "33": [["DH829", "S220"], ["DH830", "S220"], "DH831", "DH832", "DH833", "DH834", ["DH835", "S220"], "DH854", "DH855", "DH856", "DH857", "DH858", ["SOSAX", "S170+"]]
             },
-            "draw": [[]]
+            "draw": [
+                ["RARUP*", "DH230", "DH231", "DH232", "DH233", "DH234", "DH235", "DH255", "DH256", "DH257", "DH025", "ANEXI"],
+                ["RARUP*", "DH417", "DH412", "DH413", "DH414", "DH416", "DH455", "DH456", "DH457", "DH458", "DEVOD"],
+                ["RARUP*", "DH600", "DH611", "DH612", "DH613", "DH614", "DH615", "DH655", "DH656", "DH657", "DH658", "PISAS"],
+                ["RARUP*", "DH829", "DH830", "DH831", "DH832", "DH833", "DH834", "DH835", "DH854", "DH855", "DH856", "DH857", "DH858", "SOSAX"]
+            ]
         },
         "RIBSO": {
             "icao": "RIBSO",
             "name": "Ribso",
+            "suffix": {"05":"05", "15":"15", "23":"23", "33":"33"},
             "entryPoints": {
-                "L619": ["DHE", ["GOLEN", "A220"], ["OSTOR", "A120"]]
+                "RIBSO": []
             },
-            "body": [["RIBSO", "A100|S250"]],
+            "body": ["RIBSO"],
             "rwy": {
-                "05": [["DH210", "S220"], "DH211", "DH213", "DH215", "DH255", "DH256", "DH257", "DH025", "ANEXI"],
-                "15": [["DH434", "S220"], "DH435", "DH455", "DH456", "DH457", "DH458"],
-                "23": ["DH620", ["DH633", "S220"], "DH634", "DH635", "DH655", "DH656", "DH657", "DH658"],
-                "33": [["DH804", "S220"], "DH815", "DH854", "DH855", "DH856", "DH857", "DH858"]
+                "05": [["DH210", "S220"], "DH211", "DH213", "DH214", ["DH215", "S220"], "DH255", "DH256", "DH257", "DH025", ["ANEXI", "S170+"]],
```

Comment:
- `DH213` apepars to be `S220`
- Where you have `DH025`, I show `DH258`, though they appear to have the same coordinates.